src/js/util.dropchop.js

```diff
@@ -145,17 +159,28 @@ var dropchop = (function(dc) {
 
   dc.util.getBBox = function() {
     var bounds = dc.map.m.getBounds(),
-            sw = bounds.getSouthWest(),
-            ne = bounds.getNorthEast();
+        sw = bounds.getSouthWest(),
+        ne = bounds.getNorthEast();
     // we should probably check the size here?
     // node(57.7,11.9,57.8,12.0)
     return sw.lat+','+sw.lng+','+ne.lat+','+ne.lng;
   };
 
+  dc.util.getEsriBBox = function() {
+    var bounds = dc.map.m.getBounds(),
+        sw = bounds.getSouthWest(),
+        ne = bounds.getNorthEast();
+    return sw.lng+','+sw.lat+','+ne.lng+','+ne.lat;
+  };
+
   dc.util.uncollect = function(fc) {
     return fc.features[0];
   };
 
+  dc.util.esri2geo = function(arcjson) {
+    return toGeoJSON(arcjson);
+  };
```

Comment:
@alukach I should be part of the `esri2geo` package seen [here](https://github.com/cugos/dropchop/blob/master/package.json#L19). Maybe run `npm install` again?